build-system/compile/compile.js

```diff
@@ -396,19 +398,20 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
           })
           .pipe(nop())
           .on('end', resolve);
-    } else {
-      return gulp.src(srcs, {base: '.'})
-          .pipe(gulpIf(shouldShortenLicense, shortenLicense()))
-          .pipe(sourcemaps.init({loadMaps: true}))
-          .pipe(gulpClosureCompile(compilerOptionsArray))
-          .on('error', err => {
-            handleCompilerError(outputFilename);
-            reject(err);
-          })
-          .pipe(rename(outputFilename))
-          .pipe(sourcemaps.write('.'))
-          .pipe(gulp.dest(outputDir))
-          .on('end', resolve);
     }
+
+    return gulp.src(srcs, {base: '.'})
+        .pipe(gulpIf(shouldShortenLicense, shortenLicense()))
+        .pipe(sourcemaps.init({loadMaps: true}))
```

Comment:
Should `sourcemaps.init` be called before or after `babel`? Can you confirm that sourcemaps are accurate by running `gulp dist --fortesting --noextensions --full_sourcemaps` and uploading `v0.js` and `v0.js.map` to [this](http://sokra.github.io/source-map-visualization/#) sourcemaps checker?

/cc @jridgewell

Reply:
Not applying on multipass anymore so this becomes moot.